syft/messaging/message.py

```diff
@@ -156,17 +161,7 @@ def return_ids(self):
 
     @property
     def contents(self):
-        """Return a tuple with the contents of the action (backwards compatability)
-
-        Some of our codebase still assumes that all message types have a .contents attribute. However,
-        the contents attribute is very opaque in that it doesn't put any constraints on what the contents
-        might be. Since we know this message is a action, we instead choose to store contents in two pieces,
-        self.message and self.return_ids, which allows for more efficient simplification (we don't have to
-        simplify return_ids because they are always a list of integers, meaning they're already simplified)."""
-
-        message = (self.action.name, self.action.target, self.action.args, self.action.kwargs)
-
-        return (message, self.action.return_ids)
+        return self.action
```

Comment:
This might have to remain a tuple, which could either be crafted here or delegated to `self.action.contents`, depending on where we want that code to live. I'd probably rather put it on `CommandMessage`, since knowing about messaging isn't one of `Action's` responsibilities.